example_tb/core/Makefile

```diff
@@ -90,9 +89,9 @@ vlib: .lib-rtl
 	touch .lib-rtl
 
 # rebuild if we change some sourcefile
-.build-rtl: .lib-rtl $(CV_CORE_MANIFEST) $(RTLSRC_BHV) $(RTLSRC_TB_PKG) $(RTLSRC_TB)
+.build-rtl: .lib-rtl $(CV_CORE_MANIFEST) $(RTLSRC_TB_PKG) $(RTLSRC_TB)
```

Comment:
As far as I can see the following is dead Makefile code as well. If you agree, can you please remove this?

RTLSRC_INCDIR           := $(RTLSRC_HOME)/rtl/include
RTLSRC_BHV_PKG		:= $(RTLSRC_HOME)/bhv/include/cv32e40p_tracer_pkg.sv
RTLSRC_BHV_INCDIR	:= $(RTLSRC_HOME)/bhv/include
RTLSRC_PKG		:= $(addprefix $(RTLSRC_HOME)/rtl/include/,\
				cv32e40p_apu_core_pkg.sv cv32e40p_fpu_pkg.sv cv32e40p_pkg.sv)
RTLSRC			:= $(filter-out $(RTLSRC_HOME)/rtl/cv32e40p_register_file_latch.sv,\
				$(wildcard $(RTLSRC_HOME)/rtl/*.sv))